knowledge-base/configuring-iis-website-to-work-over-https.md

```diff
@@ -0,0 +1,33 @@
+---
+title:  Configuring IIS Website to Work Over HTTPS
+description: This KB article provides a few external resources for configuring IIS web site to work over HTTPS
+type: how-to
+page_title: How to deploy report server over HTTPS
+slug: configuring-iis-website-to-work-over-https
+position: 
+tags: 
+ticketid: 1416194
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Report Server</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+The Telerik Report Server is a Web application running on the IIS, and you may configure it to run under secure protocol as any other Web application. There are several approaches for configuring IIS website to work over HTTPS. You can select one depending on the version of the server.
```

Comment:
The Telerik Report Server is a Web application running on IIS server, and the configuration to run under secure protocol is the same as with any other IIS Web application. There are several approaches for configuring IIS website to work over HTTPS. Select one depending on the version of the server.